### DISCUSSION TITLE
Complete Import of Newpipe's database zip

### DISCUSSION THREAD

**Discussion by MysteriousOrb2000** [libre-tube/LibreTube#3538] [Ideas]
Newpipe is going to have a huge code refactoring soon, so they aren't going to spend much time adding new features and bug fixes. So, a feature that LibreTube could have is the ability import all information from Newpipe's database, like relevant settings, subscriptions, watch/timeline history, etc. Currently, importing Newpipe's export doesn't works for subscriptions.

**Comment by Bnyro** [comment]
That's not planned.
